pkg/config/user_config.go

```diff
@@ -285,7 +285,8 @@ type CustomCommandPrompt struct {
 	// this only applies to menuFromCommand
 	Command string `yaml:"command"`
 	Filter  string `yaml:"filter"`
-	Format  string `yaml:"format"`
+	TFormat string `yaml:"itemFormat"`
+	DFormat string `yaml:"descriptionFormat"`
```

Comment:
Can you change `TFormat` and `DFormat` to be fully written out or at least a abbreviation of the wordt you are only using the first letter of.